### PR TITLE
PRESIDECMS-626 : Added the message column on error log listing page.

### PR DESCRIPTION
**Pull request by cfmitrah**

### CODE REVIEW

File: system/services/errors/ErrorLogService.cfc [hunk 1]
```diff
@@ -54,7 +54,10 @@ component displayName="Error Log Service" {
 		var errors = [];
 
 		for( var file in files ) {
-			errors.append( { date=file.dateLastModified, filename=file.name } );
+			var errorMessage   = REMatchNoCase( "<\s*title[^>]*>(.*?)<\s*\/\s*title>", FileRead( _getLogDirectory() & "/" & file.name ) );
+			var messageContent = ReplaceList( errorMessage[1] , "<title>,</title>", "#chr(10)# ,#chr(10)#" );
```
Comment (DominicWatson) [inline system/services/errors/ErrorLogService.cfc]: We ought to be a little careful here. What if, for whatever reason, `REMatchNoCase()` returns an empty array here? 

Perhaps: 
```
var messageContent = errorMessage.len() ? ReplaceList( errorMessage[1] , "<title>,</title>", "#chr(10)# ,#chr(10)#" ) : "";
```

File: system/views/admin/errorLogs/index.cfm [hunk 2]
```diff
@@ -34,6 +35,9 @@
 							</a>
 						</td>
 						<td>
+							#left( log.message, "30" )#...<span class="fa fa-caret-down" data-rel="popover" data-trigger="hover" data-placement="left" data-content="<pre>#log.message#</pre>" data-html="true"></span>
```
Comment (DominicWatson) [inline system/views/admin/errorLogs/index.cfm]: We need to make sure that we escape the log message when inserting as an attribute in HTML:

```
... data-content="#HtmlEditFormat( '<pre>#log.message#</pre>' )#" ...
```
```

**Review by DominicWatson — CHANGES_REQUESTED** [review]
Looking good Selva, thank you. Please see the review comments - a couple of minor points.

File: system/services/errors/ErrorLogService.cfc (hunk 1 above)
Comment (cfmitrah) [reply system/services/errors/ErrorLogService.cfc]: Updated messageContent for avoid empty array.

File: system/views/admin/errorLogs/index.cfm (hunk 2 above)
Comment (cfmitrah) [reply system/views/admin/errorLogs/index.cfm]: Used HTML attribute for log message.

**Comment by DominicWatson** [comment]
Nice one, thanks @cfmitrah. Merging into upcoming release (10.8.0)

**Review by DominicWatson — APPROVED** [review]

**Pull request merged**